Add details about finer-grained log access and resetting to info logs to debug docs (#5273)
solo-io/gloo: docs/content/operations/debugging_gloo/_index.md

kubectl port-forward -n gloo-system deploy/gateway-proxy 19000:19000
```

Now you can `curl localhost:19000` and get access to the Envoy Admin API. 

> Note that after enabling `debug` logging on Envoy proxy, it does not automatically revert to the default `info` logging. In order to reset logging to the default `info` level, you will need to either use the `logging` button on the admin interface. Alternatively, you can use a single `curl` command:

Reviewer: ```suggestion
Note that after enabling `debug` logging, the Envoy proxy does not automatically revert to the default `info` logging. To reset logging to the default `info` level, you can click **logging** in the Admin UI or run the following `curl` command in the CLI:
```

kubectl logs -f -n gloo-system -l gloo=gloo
```

To follow the logs of other Gloo Edge deployments, simply change the value of the `gloo` label as shown in the table below.

| Component | Command |
| ------------- | ------------- |
| Discovery | `kubectl logs -f -n gloo-system -l gloo=discovery` |
| External Auth (Enterprise) | `kubectl logs -f -n gloo-system -l gloo=extauth` |
| Gateway | `kubectl logs -f -n gloo-system -l gloo=gateway`  |
| Gloo Control Plane | `kubectl logs -f -n gloo-system -l gloo=gloo` |
| Observability (Enterprise) | `kubectl logs -f -n gloo-system -l gloo=observability` |
| Rate Limiting (Enterprise) | `kubectl logs -f -n gloo-system -l gloo=rate-limit` |

### Changing logging levels and more

> Each Gloo Edge control plane component comes with a optional debug port that can be enabled with the `START_STATS_SERVER` environment variable. To get access to it, you can port-forward from Kubernetes like this:

Reviewer: ```suggestion
Each Gloo Edge control plane component comes with an optional debug port that you can enable with the `START_STATS_SERVER` environment variable. To get access to the port, you can forward the port of the Kubernetes deployment such as with the following command:
```

```bash
kubectl port-forward -n gloo-system deploy/gloo 9091:9091
```

> Now you can navigate to http://localhost:9091 and you get a simple page with some additional endpoints:

Reviewer: ```suggestion
Now you can navigate to `http://localhost:9091` and you get a simple page with some additional endpoints:
```

With these endpoints, you can profile the behavior of the component, adjust its logging, view the prometheus-style telemetry signals, as well as view tracing spans within the process. This is a very handy page to understand the behavior of a particular component. 

> If you'd prefer to change the log levels of individual components from the command line instead of the administrative interface, then use commands of this form.

Reviewer: ```suggestion
To change the log levels of individual Gloo Edge deployments from the CLI instead of the Admin UI, use commands similar to the following example with the `discovery` deployment.
```
